src/routes/DigitalAssets/routes/Receive/DigitalAssetsReceiveViewContainer.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import { connect } from 'react-redux'
+import { selectWalletsAddresses } from 'store/stateSelectors'
+
+import DigitalAssetsReceiveView from './DigitalAssetsReceiveView'
+
+import {
+  openView,
+  closeView,
+} from './modules/digitalAssetsReceive'
+
+const mapStateToProps = (state: AppState) => {
+  const items = selectWalletsAddresses(state).addresses.map(address => ({ address, title: '' }))
+  return {
+    items,
+  }
```

Comment:
items should be AddressNames, [use correct selector](https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/store/selectors/favorites.js#L59-L69)